### PR TITLE
[EXPLORER] Update Polish (pl-PL) translation

### PR DESCRIPTION
**Pull request by pithwz**
[EXPLORER] Update Polish (pl-PL) translation

### CODE REVIEW

File: base/shell/explorer/lang/pl-PL.rc [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * TRANSLATORS: Copyright 2007 Olaf Siejka <caemyr@gmail.com>
  *              Copyright 2009 Maciej Bialas <bjauy@o2.pl>
  *              Copyright 2017 Marcin Mikołajczak <me@m4sk.in>
- *              Copyright 2020 Piotr Hetnarowicz <piotrhwz@gmail.com>
+ *              Copyright 2023 Piotr Hetnarowicz <piotrhwz@gmail.com>
```
Comment (binarymaster) [inline base/shell/explorer/lang/pl-PL.rc:8]:
```suggestion
 *              Copyright 2020-2023 Piotr Hetnarowicz <piotrhwz@gmail.com>
```

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish

File: base/shell/explorer/lang/pl-PL.rc [hunk 2]
```diff
@@ -81,7 +81,7 @@ BEGIN
     AUTOCHECKBOX "&Trzymaj pasek zadań na wierzchu", IDC_TASKBARPROP_ONTOP, 13, 71, 200, 10
     AUTOCHECKBOX "&Grupuj podobne przyciski paska zadań", IDC_TASKBARPROP_GROUP, 13, 84, 200, 10
     AUTOCHECKBOX "Pokaż pasek &Szybkiego uruchamiania", IDC_TASKBARPROP_SHOWQL, 13, 97, 200, 10
-    AUTOCHECKBOX "U&se small icons", IDC_TASKBARPROP_SMALLICONS, 13, 110, 200, 10
+    AUTOCHECKBOX "Użyj małych ikon", IDC_TASKBARPROP_SMALLICONS, 13, 110, 200, 10
```
Comment (binarymaster) [inline base/shell/explorer/lang/pl-PL.rc:84]: Assign a mnemonic key (&).

File: base/shell/explorer/lang/pl-PL.rc [hunk 3]
```diff
@@ -104,15 +104,15 @@ CAPTION "Pasek powiadomień"
...
     AUTOCHECKBOX "&Ukryj nieaktywne ikony", IDC_TASKBARPROP_HIDEICONS, 13, 70, 125, 10
     PUSHBUTTON "&Dostosuj...", IDC_TASKBARPROP_ICONCUST, 188, 70, 50, 14
-    GROUPBOX "System icons", IDC_STATIC, 6, 99, 240, 56
-    LTEXT "Select which system icons to always show.", IDC_STATIC, 13, 113, 223, 8
+    GROUPBOX "Ikony systemowe", IDC_STATIC, 6, 99, 240, 56
+    LTEXT "Wybierz, które ikony systemowe mają być zawsze wyświetlane.", IDC_STATIC, 13, 113, 223, 8
     AUTOCHECKBOX "Pokaż &zegar", IDC_TASKBARPROP_CLOCK, 13, 127, 80, 10
     AUTOCHECKBOX "Pokaż sekundy", IDC_TASKBARPROP_SECONDS, 120, 127, 80, 10, WS_DISABLED
-    AUTOCHECKBOX "&Desktop", IDC_TASKBARPROP_DESKTOP, 13, 140, 80, 10
+    AUTOCHECKBOX "Pulpit", IDC_TASKBARPROP_DESKTOP, 13, 140, 80, 10
```
Comment (binarymaster) [inline base/shell/explorer/lang/pl-PL.rc:115]: Same here.

**Review by Saibamen — APPROVED** [review]

**Pull request closed without merging**